### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by c3n8fy**

### CODE REVIEW

**Comment by c3n8fy** [comment]
Fixed very critical bugs. Now all bugs are eliminated.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```
Comment (CSK7KO) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Wouldn't a talkative name be better? Something like `randomGenerator`.
Why isn't it static? Couldn't instances share it?

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (CSK7KO) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: The message in the exception should tell what's wrong with the argument.

**Review by CSK7KO — CHANGES_REQUESTED** [review]
Just some cosmetics and good to go.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (c3n8fy) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: thanks, I'll fix it :)

**Review by CSK7KO — APPROVED** [review]

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 2 above)
Comment (c3n8fy) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: ok!

**Pull request merged**